code/go/0chain.net/smartcontract/dbs/event/delegate_pool.go

```diff
@@ -26,6 +28,7 @@ type DelegatePool struct {
 	Status               spenum.PoolStatus `json:"status" gorm:"index:idx_dprov_active,priority:3;index:idx_ddel_active,priority:3;index:idx_dp_total_staked,priority:2"`
 	RoundCreated         int64             `json:"round_created"`
 	RoundPoolLastUpdated int64             `json:"round_pool_last_updated"`
+	StakedAt             common.Timestamp  `json:"staked_at"`
```

Comment:
why doesn't createdAt work?

Reply:
Peterlimg instructed me to use txn.CreationDate's value for stakedAt instead of using the system's time when the entity is inserted into the db

Reply:
@dabasov the `StakedAt` has already used in [DelegatePool](https://github.com/0chain/0chain/blob/02dd5f44df971276225f201d5e1f4ae576751069/code/go/0chain.net/smartcontract/stakepool/stakepool.go#L70), so should be fine to use it to keep consistent.

Reply:
do we have 2 columns that keeps the same data? I mean does this two columns `CreatedAt` and `StakedAt` really  differ?

Reply:
The CreatedAt column of &event.DelegatePool differs from StakedAt column of the same struct, cos CreatedAt is when DelegatePool entity is persisted in DB, while StakedAt is when gb is created.

The &stakepool.DelegatePool's StakedAt value is used for &event.DelegatePool's StakedAt value. This is for maintaining consistency.

I'm new to this & my understanding may be incorrect.
@peterlimg, @Kishan-Dhakan Could you please correct me if I'm wrong or help improve this answer?

Reply:
you are right on how they work, but I mean does it make any difference which use, they should be close, maybe we can put this timestamp directly to CreatedAt, or remove CreatedAt from this table, I don't think that we use both this values anywhere

Reply:
Maybe we can replace model.UpdatableModel(containing ID, CreatedAt & UpdatedAt) in DelegatePool with just ID, eliminating both CreatedAt & UpdatedAt columns.
Should I do this?
@peterlimg @Kishan-Dhakan @dabasov
One concern is that we may need CreatedAt & UpdatedAt for debugging purpose in future